bedrock/products/templates/products/vpn/variations/cta-b.html

```diff
@@ -25,7 +25,7 @@
     }
     ) }}
     <p>
-      <a href="#pricing" data-cta-type="button"data-cta-position="primary"data-cta-text="Scroll to pricing">View all pricing plans</a>
+     *For a 12-month plan. <a href="#pricing" data-cta-type="button"data-cta-position="primary"data-cta-text="Scroll to pricing">View all pricing plans</a>
```

Comment:
Nit: add a space after the *.
And there should be spaces between the attributes.
```suggestion
     * For a 12-month plan. <a href="#pricing" data-cta-type="button" data-cta-position="primary" data-cta-text="Scroll to pricing">View all pricing plans</a>
```